BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb

```diff
@@ -108,7 +108,7 @@
   <ul class="tsk-prm-list">
     <li>
       <%%
-        input.(id, name,
+        input.(id, name+"[]",
```

Comment:
Just a general note. I wasn't sure if the input element was to do double duty for single values and for array values; if it's supposed to always be array values, then all's fine.